client/knowledge_base_search/src/containers/ViewAllContainer/ViewAllContainer.jsx

```diff
@@ -147,6 +148,7 @@ ViewAllContainer.propTypes = {
     is_training_query: bool,
   })).isRequired,
   questionsPerPage: number.isRequired,
+  selectedFeature: string,
```

Comment:
I understand that the container component for contains `selectedFeature` in state as a string, but it seems like this component only cares if this is specifically `trained`. I think that this component's prop makes more sense as a bool called something like `isTrained`.